site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -336,12 +336,12 @@ export const WorkspaceReadyPage: FC<WorkspaceReadyPageProps> = ({
           setIsConfirmingUpdate(false);
         }}
         onClose={() => setIsConfirmingUpdate(false)}
-        title="Update and restart?"
+        title="Update workspace?"
         confirmText="Update"
         description={
           <Stack>
             <p>
-              Restarting your workspace will stop all running processes and{" "}
+              Updating your workspace will stop all running processes and{" "}
```

Comment:
```suggestion
              Updating your workspace will{" "}
```
`stop all running processes` is kind of misleading, right?

Reply:
You are right. I am having some trouble trying to find the difference between our phrasing of "updating" vs the actual terraform actions :thinking: